EIPS/eip-4512.md

```diff
@@ -0,0 +1,168 @@
+---
+eip: 4512
+title: Non-Fungible Token with usage rights
+description: Separation of Non-Fungible Token ownership and usage rights , Truly in line with real life.
+status: Draft
+type: Standards Track
+author: KKimos (@KKimos), KKimos <bol@zju.edu.cn>
+category: ERC
+created: 2021-12-01
+requires: 165, 721
+---
+
+## Simple Summary
+Only add a user role, change the way NFT sharing.
+
+## Abstract
+
+This standard adds a new right of Non-Fungible Token that the right to use. Through this standard, you can achieve :
+
+- Separation of the right to use and ownership of Non-Fungible Token
+- Non-secured lease Non-Fungible Token
+- You can continue to use it after you mortgage the Non-Fungible Token
+- Metaverse sharing economy
+
+It is precisely because of the separation of ownership and use right that the utilization rate of assets can be greater. You must distinguish between the rights of the user and the owner.
+
+## Motivation
+
+Uber, airbnb, and WeWork promote the development of the sharing economy. In the blockchain , the ERC721-based leasing system has encountered great bottlenecks. For example, excess assets need to be mortgaged for leasing, The increase in NFT prices will lead to defaults ,You cannot continue to use it during NFT in mortgages. As a result, a new NFT standard is proposed, which is fully compatible with ERC721. Separate the right to use and own the NFT.
```

Comment:
Airbnb

Reply:
Thank you for your suggestion, I have revised it.